Bottom: 320391e09eff895944f656162aa0e4b0149e78c5
Top:    36966c4401d831ff9b2904957aa148db5d4281a8
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-18 17:14:56 -0700

Refresh of no-more-reset

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
index 0b7ab21d521..128270ceef9 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.h
@@ -753,7 +753,7 @@ llvm::Expected<PythonObject> runStringMultiLine(CStringArg string,
 /* Sometimes the best way to interact with a python interpreter is
  * to run some python code.   You construct a PythonScript with
  * script string and a function name, and you get a C++ callable
- * objedct that calls the python function.
+ * object that calls the python function.
  *
  * Example:
  *
